Presentation.Web/Tests/Helpers/InterfaceHelper.ts

```diff
@@ -0,0 +1,88 @@
+﻿import CSSLocator = require("../object-wrappers/CSSLocatorHelper");
+import Select2 = require("./Select2Helper");
+import constant = require("../Utility/Constants")
+
+class InterfaceHelper {
+    private const = new constant();s
+    private cssHelper = new CSSLocator();
+
+    public writeDataToAllInputs(data: string, exposedBy: string, sysInterface: string, access: string, belongsTo: string, dataTypeTable: string) {
+        return this.writeDataToTextInput(data, this.const.interfaceNameInput)
+            .then(() => {
+                return this.writeDataToTextInput(data, this.const.interfaceIdInput);
+            }).then(() => {
+                return this.writeDataToTextInput(data, this.const.interfaceVersionInput);
+            }).then(() => {
+                return this.writeDataToTextInput(data, this.const.interfaceDescriptionInput);
+            }).then(() => {
+                return this.writeDataToTextInput(data, this.const.interfaceDescriptionLinkInput);
+            }).then(() => {
+                return this.writeDataToTextInput(data, this.const.interfaceNoteInput);
+            }).then(() => {
+                return this.selectDataFromSelect2Field(exposedBy, this.const.interfaceSelectExhibit);
+            }).then(() => {
+                return this.selectDataFromSelect2Field(sysInterface, this.const.interfaceSelectInterface);
+            }).then(() => {
+                return this.selectDataFromSelect2Field(access, this.const.interfaceSelectAccess);
+            }).then(() => {
+                return this.selectDataFromSelect2Field(belongsTo, this.const.interfaceSelectBelongs);
+            }).then(() => {
+                return this.writeDataToTable(data, dataTypeTable);
+            });
+    }
+
+    public verifyDataWasSaved(data: string, exposedBy: string, sysInterface: string, access: string, belongsTo: string, dataTypeTable: string) {
+        // text 
+        this.verifyDataFromTextInput(data, this.const.interfaceNameInput);
```

Comment:
lidt synd at det er samme tekst du indsætter i alle felter. Kan du ikke sørge for at det er forskelligt?